training/models/training_action.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+
+from openerp import _, api, fields, models
+from .common import M
+from .. import exceptions as ex
+
+
+class ActionType(models.Model):
+    """Types of training actions.
+
+    Depending on the training action type, a training action may expect some
+    type of hours. For example:
+
+    - If a training action's type is "on-site", it expects on-site hours.
+    - If its type is "online", it expects online hours.
+    - If its type is "mixed", it expects both on-site and online hours.
```

Comment:
It could happen if `expected_duration_type_ids` were a Selection field, but being a Many2many what you say is not possible.